bitseq/sequence.go

```diff
@@ -240,8 +251,11 @@ func (h *Handle) set(ordinal uint32, any bool, release bool) (uint32, error) {
 			bytePos, bitPos = ordinalToPos(ordinal)
 		} else {
 			if any {
-				bytePos, bitPos, err = getFirstAvailable(h.head)
+				bytePos, bitPos, err = getFirstAvailable(h.head, start)
 				ret = posToOrdinal(bytePos, bitPos)
+				if end < ret {
```

Comment:
not a comment. but mostly a question. 
For the case of `(h *Handle) Set(ordinal uint32) error {`, the start and end are 0.
does `posToOrdinal` for this case return 0 ?
If not, this is obviously going to fail. can you please confirm ?